### PR TITLE
Added new skeleton outline

### PR DESCRIPTION
**Pull request by favour-chibueze**

### CODE REVIEW

**Comment by favour-chibueze** [comment]
@RCheesley Please, review this PR. As I mentioned to you earlier, the Vale checks are failing on the Twilio links

**Review by RCheesley — CHANGES_REQUESTED** [review]
Looks good, just one small thing that I noticed which I think is an over-reactive Vale warning! Great work @favour-chibueze 🚀

File: docs/getting_started/how_to_update_mautic.rst [hunk 1]
```diff
@@ -115,7 +115,7 @@ If you need help, you can ask for it in several places. You should remember that
 
 - The :xref:`Mautic Community Forums` is the place where you can ask questions about your configuration if you think it is the cause of the problem. Please search before posting your question, since someone may have already answered it.
 
-- The live :xref:`Mautic Community Slack` is also available, but all support requests have to be posted on the forums. Post there first, then drop a link in Slack if you plan to discuss it there.
+- The live :xref:`Mautic Community Slack` is also available, but all support requests have to be posted on the forums. POST there first, then drop a link in Slack if you plan to discuss it there.
```
[the remark this answers is not in the document]
Comment (favour-chibueze) [reply docs/getting_started/how_to_update_mautic.rst]: It's been resolved @RCheesley

**Review by RCheesley — APPROVED** [review]
Rock and roll, LGTM! 🎸 🚀

**Pull request merged**